### PR TITLE
Update directory structure and package name

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Related issue: [1623](https://github.com/ballerina-platform/ballerina-standard-library/issues/1623)
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -70,8 +65,8 @@ subprojects {
     }
     dependencies {
         /* Standard libraries */
-        ballerinaStdLibs "org.ballerinalang:task-ballerina:${stdlibTaskVersion}"
-        ballerinaStdLibs "org.ballerinalang:time-ballerina:${stdlibTimeVersion}"
+        ballerinaStdLibs "io.ballerina.stdlib:task-ballerina:${stdlibTaskVersion}"
+        ballerinaStdLibs "iio.ballerina.stdlib:time-ballerina:${stdlibTimeVersion}"
```
Comment (ThisaruGuruge) [inline build.gradle]:
```suggestion
        ballerinaStdLibs "io.ballerina.stdlib:time-ballerina:${stdlibTimeVersion}"
```

File: build.gradle [hunk 2]
```diff
@@ -16,17 +16,12 @@
...
+    id "net.researchgate.release" version "${researchgateReleaseVersion}"
 }
 
-ext.ballerinaLangVersion = project.ballerinaLangVersion
-ext.stdlibTaskVersion = project.stdlibTaskVersion
-ext.stdlibTimeVersion = project.stdlibTimeVersion
-ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
-
 allprojects {
     group = 'org.ballerinalang'
```
Comment (MadhukaHarith92) [inline build.gradle:26]:
```suggestion
    group = project.group
```

File: gradle.properties [hunk 3]
```diff
@@ -1,9 +1,15 @@
 org.gradle.caching=true
-group=org.ballerinalang
+group=io.ballerina.stdlib
 version=2.1.0-beta.3-SNAPSHOT
 ballerinaLangVersion=2.0.0-beta.2.1
 stdlibTaskVersion=2.0.0-beta.2
 stdlibTimeVersion=2.0.0-beta.2
```
Comment (ThisaruGuruge) [inline gradle.properties:6]: These versions should be updated

**Review by ThisaruGuruge — COMMENTED** [review]
Stdlib module versions should be updated from the version bump PR, and then the group IDs should be updated in the build.gradle file.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**